luma/core/virtual.py

```diff
@@ -227,8 +229,14 @@ def println(self, text=""):
         Prints the supplied text to the device, scrolling where necessary.
         The text is always followed by a newline.
         """
-        self.puts(text)
-        self.newline()
+        if self.word_wrap:
+            text = wrap(text, width=self.width)
```

Comment:
fixed it!

![ansi_colors_wrapped](https://cloud.githubusercontent.com/assets/305679/25097051/2dc9d1aa-23a3-11e7-8a9f-48f9cc4d00d7.png)

@rm-hull also used version added `0.9.0`, I assume that'll be the next version?